rtsp.go

```diff
@@ -148,8 +157,38 @@ func (rc *rtspCamera) reconnectClient() (err error) {
 		return err
 	}
 
-	// find the H264 media and format
+	// find codec info from GetStreamInfo
+	codecInfo := GetStreamInfo(rc.u.String())
```

Comment:
Always think about whether the functions, variables and constants you're exporting in Go actually need to be exported.

Reply:
Good call out.